packages/core/src/modules/message-pickup/MessagePickupApi.ts

```diff
@@ -255,18 +257,19 @@ export class MessagePickupApi<MPPs extends MessagePickupProtocol[] = [V1MessageP
    */
   public async setLiveDeliveryMode(options: SetLiveDeliveryModeOptions): Promise<SetLiveDeliveryModeReturnType> {
     const connectionRecord = await this.connectionService.getById(this.agentContext, options.connectionId)
-
     const protocol = this.getProtocol(options.protocolVersion)
     const { message } = await protocol.setLiveDeliveryMode(this.agentContext, {
       connectionRecord,
       liveDelivery: options.liveDelivery,
     })
 
+    // Live mode requires a long-lived transport session, so we'll require WebSockets to send this message
```

Comment:
Not for now, but would be good if we can configure this on the transport class, so we can add the requirement here and filter dynamicslly